client/sites/settings/site/privacy/notice.tsx

```diff
@@ -56,10 +83,70 @@ const SiteSettingPrivacyNotice = ( { selectedSite, siteSlug }: SiteSettingPrivac
 					>
 						{ translate( 'Upgrade' ) }
 					</Button>
-				</div>
-			</div>
-		</>
+				</>
+			}
+		/>
+	);
+};
+
+interface SiteSettingsPrivacyDiscourageSearchEnginesNoticeProps {
+	selectedSite: SiteDetails | null | undefined;
+	siteSlug: string | null;
+}
+
+export const SiteSettingsPrivacyDiscourageSearchEnginesNotice = ( {
+	selectedSite,
+	siteSlug,
+}: SiteSettingsPrivacyDiscourageSearchEnginesNoticeProps ) => {
+	const translate = useTranslate();
+	const hasNonWpcomDomains =
+		useSelector( ( state ) => getDomainsBySiteId( state, selectedSite?.ID ) || [] ).filter(
+			( domain ) => ! domain.isWPCOMDomain
+		).length > 0;
+
+	return (
+		<SiteSettingsAdvancedCustomizationNotice
+			notice={
+				<span>
+					{ translate(
+						'Your site is using {{strong}}wpcomstaging.com{{/strong}} as the primary domain, which is prevented from being indexed by search engines. To change this, set a custom domain as the primary domain.',
+						{
+							components: {
+								strong: <strong />,
+							},
+						}
+					) }
+				</span>
+			}
+			actions={
+				hasNonWpcomDomains ? (
+					<Button
+						className="is-primary"
+						href={ addQueryArgs( `/domains/manage/${ siteSlug }`, {
+							source: SOURCE_SETTINGS_ADMINISTRATION,
+						} ) }
+						onClick={ () =>
+							recordTracksEvent( 'calypso_settings_site_privacy_manage_domains_button_click' )
+						}
+					>
+						{ translate( 'Manage domains' ) }
+					</Button>
+				) : (
+					<Button
+						className="is-primary"
+						href={ addQueryArgs( `/domains/add/${ siteSlug }`, {
+							redirect_to: window.location.pathname,
+						} ) }
+						onClick={ () =>
+							recordTracksEvent( 'calypso_settings_site_privacy_add_domain_button_click' )
+						}
+					>
+						{ translate( 'Add new domain' ) }
+					</Button>
+				)
+			}
+		/>
 	);
 };
 
-export default SiteSettingPrivacyNotice;
+// When an AT site is still using the .wpcomstaging.com address, it has a special robots.txt file on it.  Note that wpcomstaging.com domains are prevented from search engine indexing by default.
```

Comment:
Is it expected to have this comment at the end of the file? 🤔

Reply:
oops forgot to remove it 😓